src/services/ApiCalls.tsx

```diff
@@ -0,0 +1,63 @@
+import { URL_SERVER } from "constants/serverConnexion";
+
+const calculateAge = (birthday: any) => {
+  var ageDifMs = Date.now() - birthday;
+  var ageDate = new Date(ageDifMs); // miliseconds from epoch
+  return Math.abs(ageDate.getUTCFullYear() - 1970);
+};
+interface Patient {
+  firstName: String;
+  lastName: String;
+  age: number;
+  id: String;
+}
+
+export const CallApi = (typeR: string, queryR: any) => {
+  let mkFhir = require("fhir.js");
```

Comment:
Is this an import?
If so, why isn't it done at the top of the file. And could you use the `import` syntax rather than `require`?

Reply:
import {fhir.js} from "fhir.js" is not working, but I moved it outside CallApi function to avoid loading it at each call.